### ISSUE TITLE
Peer Review by Arash Shamseddini

### ISSUE THREAD

**Issue by arashshams** [UBC-MDS/DSCI522_group17#25]
Hi Team,
Great job with your project and congrats. I really liked what you guys did with your repo structuring. 
Below you can find my feedback based on Release 0.2.0 of your work:
• Documentation: Like I said, I think your project documentation is well designed and I did not have trouble finding my way around your repository. The folder names make sense as well.
• Code: All is good, and I was able to reproduce almost the same results. However, the “random_state” parameter should have been taken into account for the sake of exact reproducibility. Also, I think it would not hurt if you pointed out to that fact that “slightly different fit/score times” will be obtained when it comes to reproducibility; maybe a simple disclaimer. 
• Analysis and reasoning: Your analysis and reasoning is in overall concrete and I did not have much trouble with understanding what the workflow is in your analysis and where you are headed. In the “Results” section, you have provided good reasoning with why you have picked certain plots and what you are aiming to get out of each one. However, maybe you could elaborate a little more on this section of the report, in why you picked the models you picked. Also, if I were you, maybe I would plot my final results of the performance of the models instead of tabulating them. Finally, I found the second paragraph of the “Analysis” section redundant information. Maybe that is where you could explain why you chose these models. Don’t you guys think that maybe you should list the features in your analysis before you present the plots in the “Results” section? This way the reader knows what is what with your x and y axes of the plots.
• Communication: Your language is concise and clear and I think the whole work is well communicated. Good choice of plots has made it interactive too.
• Small not that big of a deal suggestion: I would change the date in the header of the “README” to something like Nov. – Dec. 2020. 
In a nutshell, I think you have done an amazing job with your project and wish you all the best.

**Comment by jachang0628** [comment]
Thank you for your feedback! We addressed the issues regarding reproducibility.
